tool/tsh/tsh.go

```diff
@@ -1448,7 +1448,27 @@ func onLogin(cf *CLIConf) error {
 	cf.Proxy = webProxyHost
 
 	// Print status to show information of the logged in user.
-	return trace.Wrap(onStatus(cf))
+	if err := onStatus(cf); err != nil {
+		return trace.Wrap(err)
+	}
+
+	if err := displayLicenseWarnings(cf.Context, tc); err != nil {
+		log.WithError(err).Debug("Failed to display license warnings.")
+	}
+
+	return nil
+}
+
+// displayLicenseWarnings displays license out of compliance warnings.
+func displayLicenseWarnings(ctx context.Context, tc *client.TeleportClient) error {
+	resp, err := tc.Ping(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	for _, warning := range resp.LicenseWarnings {
+		fmt.Fprintln(os.Stderr, warning)
+	}
+	return nil
 }
```

Comment:
The warning message is not delimited by a new line https://github.com/gravitational/houston/blob/bddea97761a32679c9a3bec358da3f9f3698b5a6/lib/web/apiserver.go#L373-L383

I'll add an extra newline so that there is always space between two warning messages. 

I implemented it this way to allow for multiple warning messages, but I'm thinking we should try to avoid having more than one warning message displayed this way. If we do want to display more notifications this way, we should probably put some more consideration into how it should look.